srcpkgs/NetworkManager/patches/no-gtk-doc.patch

```diff
@@ -0,0 +1,28 @@
+Index: b/meson.build
+===================================================================
+--- a/meson.build
++++ b/meson.build
+@@ -943,10 +943,8 @@ if enable_qt
+ endif
+ 
+ if enable_docs
+-  assert(enable_introspection, '-Ddocs=true requires -Dintrospection=true')
+   assert(meson.version().version_compare('>= 0.46.0'), '-Ddocs requires meson >= 0.46')
+   subdir('man')
+-  subdir('docs')
```

Comment:
I don't get this patch, doesn't it make more sense to make doc generation depend on the gir build option? Or just always build with introspection enabled.

Reply:
`docs` subdir is about building gtk-doc, which is meant for developer and requires `gir`.
Normal users want man instead.
I couldn't build the gtk-doc on my _single_ try, so I disabled it. No code, no problem :-p

Reply:
Ah, I see! I'd appreciate a comment on top that we want only man, not the full development docs, then.

And maybe make a request to upstream for it?

Reply:
Would it be appropriate to change `if enable_docs` to `if enable_docs and enable_introspection` and get rid of the rest of the changes in this section?

Reply:
I filed an issue: https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/issues/689

Reply:
According to discussion in the link above, it seems like their test process grows exponentially more complex the more options are added, and a hypothetical `-Dman` option might not get tested at all. If we really don't want to ship the developments docs, would it be better to build it and simply remove the directory in a post-install step?

Reply:
I think the issue was that building the docs failed, while the man pages worked.

Reply:
For the record, this isn't a blocker for merging the PR.

Reply:
Building fails on ARM architectures without the patch, but builds successfully on all architectures with the patch.

Reply:
Was that the expected outcome, specifically with ARM archs? I guess that's not what I was expecting. If it was, I'll go ahead and push the other commits and mark this PR ready for review.

Reply:
Kinda makes sense, it'd be cross builds that'd be affected by it.